<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.8.5">
  <compounddef id="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception" kind="class" prot="public">
    <compoundname>org::apache::commons::math::FunctionEvaluationException</compoundname>
    <basecompoundref refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception" prot="public" virt="non-virtual">org.apache.commons.math.MathException</basecompoundref>
    <derivedcompoundref refid="classorg_1_1apache_1_1commons_1_1math_1_1_argument_outside_domain_exception" prot="public" virt="non-virtual">org.apache.commons.math.ArgumentOutsideDomainException</derivedcompoundref>
      <sectiondef kind="private-static-attrib">
      <memberdef kind="variable" id="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a430040ad99f0d58fdcc50fe27516dac0" prot="private" static="yes" mutable="no">
        <type>final long</type>
        <definition>final long org.apache.commons.math.FunctionEvaluationException.serialVersionUID</definition>
        <argsstring></argsstring>
        <name>serialVersionUID</name>
        <initializer>= -7619974756160279127L</initializer>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para>Serializable version identifier. </para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" line="30" column="1" bodyfile="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" bodystart="30" bodyend="-1"/>
      </memberdef>
      </sectiondef>
      <sectiondef kind="private-attrib">
      <memberdef kind="variable" id="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a351714d5339903e021be7a39e2adca53" prot="private" static="no" mutable="no">
        <type>double</type>
        <definition>double org.apache.commons.math.FunctionEvaluationException.argument</definition>
        <argsstring></argsstring>
        <name>argument</name>
        <initializer>= Double.NaN</initializer>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para>Argument causing function evaluation failure </para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" line="33" column="1" bodyfile="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" bodystart="33" bodyend="-1"/>
      </memberdef>
      </sectiondef>
      <sectiondef kind="public-func">
      <memberdef kind="function" id="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1aa34ff769fd5096cac91413650835d4f2" prot="public" static="no" const="no" explicit="no" inline="yes" virt="non-virtual">
        <type></type>
        <definition>org.apache.commons.math.FunctionEvaluationException.FunctionEvaluationException</definition>
        <argsstring>(double argument)</argsstring>
        <name>FunctionEvaluationException</name>
        <param>
          <type>double</type>
          <declname>argument</declname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para>Construct an exception indicating the argument value that caused the function evaluation to fail.</para><para><parameterlist kind="param"><parameteritem>
<parameternamelist>
<parametername>argument</parametername>
</parameternamelist>
<parameterdescription>
<para>the failing function argument </para></parameterdescription>
</parameteritem>
</parameterlist>
</para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" line="41" column="1" bodyfile="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" bodystart="41" bodyend="45"/>
      </memberdef>
      <memberdef kind="function" id="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1ac6a0fb990d010b9aa3919456478778d5" prot="public" static="no" const="no" explicit="no" inline="yes" virt="non-virtual">
        <type></type>
        <definition>org.apache.commons.math.FunctionEvaluationException.FunctionEvaluationException</definition>
        <argsstring>(double argument, String message)</argsstring>
        <name>FunctionEvaluationException</name>
        <param>
          <type>double</type>
          <declname>argument</declname>
        </param>
        <param>
          <type>String</type>
          <declname>message</declname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para>Construct an exception using the given argument and message text.</para><para><parameterlist kind="param"><parameteritem>
<parameternamelist>
<parametername>argument</parametername>
</parameternamelist>
<parameterdescription>
<para>the failing function argument </para></parameterdescription>
</parameteritem>
<parameteritem>
<parameternamelist>
<parametername>message</parametername>
</parameternamelist>
<parameterdescription>
<para>the exception message text </para></parameterdescription>
</parameteritem>
</parameterlist>
<xrefsect id="deprecated_1_deprecated000067"><xreftitle>Deprecated</xreftitle><xrefdescription><para>as of 1.2, replaced by <ref refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a2bfa27be0ddaaaa744b9c2a7f83c5f89" kindref="member">FunctionEvaluationException(double, String, Object[])</ref> </para></xrefdescription></xrefsect></para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" line="55" column="1" bodyfile="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" bodystart="55" bodyend="58"/>
      </memberdef>
      <memberdef kind="function" id="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a2bfa27be0ddaaaa744b9c2a7f83c5f89" prot="public" static="no" const="no" explicit="no" inline="yes" virt="non-virtual">
        <type></type>
        <definition>org.apache.commons.math.FunctionEvaluationException.FunctionEvaluationException</definition>
        <argsstring>(double argument, String pattern, Object[] arguments)</argsstring>
        <name>FunctionEvaluationException</name>
        <param>
          <type>double</type>
          <declname>argument</declname>
        </param>
        <param>
          <type>String</type>
          <declname>pattern</declname>
        </param>
        <param>
          <type>Object[]</type>
          <declname>arguments</declname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para>Constructs an exception with specified formatted detail message. Message formatting is delegated to <ref refid="" kindref="compound">java.text.MessageFormat</ref>. <parameterlist kind="param"><parameteritem>
<parameternamelist>
<parametername>argument</parametername>
</parameternamelist>
<parameterdescription>
<para>the failing function argument </para></parameterdescription>
</parameteritem>
<parameteritem>
<parameternamelist>
<parametername>pattern</parametername>
</parameternamelist>
<parameterdescription>
<para>format specifier </para></parameterdescription>
</parameteritem>
<parameteritem>
<parameternamelist>
<parametername>arguments</parametername>
</parameternamelist>
<parameterdescription>
<para>format arguments </para></parameterdescription>
</parameteritem>
</parameterlist>
<simplesect kind="since"><para>1.2 </para></simplesect>
</para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" line="68" column="1" bodyfile="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" bodystart="68" bodyend="72"/>
      </memberdef>
      <memberdef kind="function" id="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a74156e50005b1d7e243b7e797011fcf0" prot="public" static="no" const="no" explicit="no" inline="yes" virt="non-virtual">
        <type></type>
        <definition>org.apache.commons.math.FunctionEvaluationException.FunctionEvaluationException</definition>
        <argsstring>(double argument, String message, Throwable cause)</argsstring>
        <name>FunctionEvaluationException</name>
        <param>
          <type>double</type>
          <declname>argument</declname>
        </param>
        <param>
          <type>String</type>
          <declname>message</declname>
        </param>
        <param>
          <type>Throwable</type>
          <declname>cause</declname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para>Construct an exception with the given argument, message and root cause.</para><para><parameterlist kind="param"><parameteritem>
<parameternamelist>
<parametername>argument</parametername>
</parameternamelist>
<parameterdescription>
<para>the failing function argument </para></parameterdescription>
</parameteritem>
<parameteritem>
<parameternamelist>
<parametername>message</parametername>
</parameternamelist>
<parameterdescription>
<para>descriptive error message </para></parameterdescription>
</parameteritem>
<parameteritem>
<parameternamelist>
<parametername>cause</parametername>
</parameternamelist>
<parameterdescription>
<para>root cause. </para></parameterdescription>
</parameteritem>
</parameterlist>
<xrefsect id="deprecated_1_deprecated000068"><xreftitle>Deprecated</xreftitle><xrefdescription><para>as of 1.2, replaced by <ref refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1ae4059cbef0b27fabfc0973803fdea56d" kindref="member">FunctionEvaluationException(double, String, Object[], Throwable)</ref> </para></xrefdescription></xrefsect></para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" line="82" column="1" bodyfile="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" bodystart="82" bodyend="86"/>
      </memberdef>
      <memberdef kind="function" id="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a4d0053c292a35004a3d3d3362011cc3e" prot="public" static="no" const="no" explicit="no" inline="yes" virt="non-virtual">
        <type></type>
        <definition>org.apache.commons.math.FunctionEvaluationException.FunctionEvaluationException</definition>
        <argsstring>(double argument, Throwable cause)</argsstring>
        <name>FunctionEvaluationException</name>
        <param>
          <type>double</type>
          <declname>argument</declname>
        </param>
        <param>
          <type>Throwable</type>
          <declname>cause</declname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para>Constructs an exception with specified root cause. Message formatting is delegated to <ref refid="" kindref="compound">java.text.MessageFormat</ref>. <parameterlist kind="param"><parameteritem>
<parameternamelist>
<parametername>argument</parametername>
</parameternamelist>
<parameterdescription>
<para>the failing function argument </para></parameterdescription>
</parameteritem>
<parameteritem>
<parameternamelist>
<parametername>cause</parametername>
</parameternamelist>
<parameterdescription>
<para>the exception or error that caused this exception to be thrown </para></parameterdescription>
</parameteritem>
</parameterlist>
<simplesect kind="since"><para>1.2 </para></simplesect>
</para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" line="95" column="1" bodyfile="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" bodystart="95" bodyend="98"/>
      </memberdef>
      <memberdef kind="function" id="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1ae4059cbef0b27fabfc0973803fdea56d" prot="public" static="no" const="no" explicit="no" inline="yes" virt="non-virtual">
        <type></type>
        <definition>org.apache.commons.math.FunctionEvaluationException.FunctionEvaluationException</definition>
        <argsstring>(double argument, String pattern, Object[] arguments, Throwable cause)</argsstring>
        <name>FunctionEvaluationException</name>
        <param>
          <type>double</type>
          <declname>argument</declname>
        </param>
        <param>
          <type>String</type>
          <declname>pattern</declname>
        </param>
        <param>
          <type>Object[]</type>
          <declname>arguments</declname>
        </param>
        <param>
          <type>Throwable</type>
          <declname>cause</declname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para>Constructs an exception with specified formatted detail message and root cause. Message formatting is delegated to <ref refid="" kindref="compound">java.text.MessageFormat</ref>. <parameterlist kind="param"><parameteritem>
<parameternamelist>
<parametername>argument</parametername>
</parameternamelist>
<parameterdescription>
<para>the failing function argument </para></parameterdescription>
</parameteritem>
<parameteritem>
<parameternamelist>
<parametername>pattern</parametername>
</parameternamelist>
<parameterdescription>
<para>format specifier </para></parameterdescription>
</parameteritem>
<parameteritem>
<parameternamelist>
<parametername>arguments</parametername>
</parameternamelist>
<parameterdescription>
<para>format arguments </para></parameterdescription>
</parameteritem>
<parameteritem>
<parameternamelist>
<parametername>cause</parametername>
</parameternamelist>
<parameterdescription>
<para>the exception or error that caused this exception to be thrown </para></parameterdescription>
</parameteritem>
</parameterlist>
<simplesect kind="since"><para>1.2 </para></simplesect>
</para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" line="109" column="1" bodyfile="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" bodystart="109" bodyend="114"/>
      </memberdef>
      <memberdef kind="function" id="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a28d8e7cbca775686a170215320c79b7a" prot="public" static="no" const="no" explicit="no" inline="yes" virt="non-virtual">
        <type>double</type>
        <definition>double org.apache.commons.math.FunctionEvaluationException.getArgument</definition>
        <argsstring>()</argsstring>
        <name>getArgument</name>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para>Returns the function argument that caused this exception.</para><para><simplesect kind="return"><para>argument that caused function evaluation to fail </para></simplesect>
</para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" line="121" column="1" bodyfile="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" bodystart="121" bodyend="123"/>
      </memberdef>
      </sectiondef>
    <briefdescription>
    </briefdescription>
    <detaileddescription>
<para>Exception thrown when an error occurs evaluating a function. </para><para>Maintains an <computeroutput>argument</computeroutput> property holding the input value that caused the function evaluation to fail.</para><para><simplesect kind="version"><para></para></simplesect>
<simplesect kind="rcs"><title>Revision:</title><para>620312 </para></simplesect>
<simplesect kind="rcs"><title>Date:</title><para>2008-02-10 12:28:59 -0700 (Sun, 10 Feb 2008) </para></simplesect>
</para>    </detaileddescription>
    <inheritancegraph>
      <node id="8059">
        <label>org.apache.commons.math.FunctionEvaluationException</label>
        <link refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception"/>
        <childnode refid="8060" relation="public-inheritance">
        </childnode>
      </node>
      <node id="8061">
        <label>Exception</label>
      </node>
      <node id="8062">
        <label>org.apache.commons.math.ArgumentOutsideDomainException</label>
        <link refid="classorg_1_1apache_1_1commons_1_1math_1_1_argument_outside_domain_exception"/>
        <childnode refid="8059" relation="public-inheritance">
        </childnode>
      </node>
      <node id="8060">
        <label>org.apache.commons.math.MathException</label>
        <link refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception"/>
        <childnode refid="8061" relation="public-inheritance">
        </childnode>
      </node>
    </inheritancegraph>
    <collaborationgraph>
      <node id="8063">
        <label>org.apache.commons.math.FunctionEvaluationException</label>
        <link refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception"/>
        <childnode refid="8064" relation="public-inheritance">
        </childnode>
      </node>
      <node id="8065">
        <label>Exception</label>
      </node>
      <node id="8064">
        <label>org.apache.commons.math.MathException</label>
        <link refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception"/>
        <childnode refid="8065" relation="public-inheritance">
        </childnode>
      </node>
    </collaborationgraph>
    <location file="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" line="27" column="1" bodyfile="C:/Users/JulyaMaria/Desktop/Desmo-J/org/apache/commons/math/FunctionEvaluationException.java" bodystart="27" bodyend="125"/>
    <listofallmembers>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1ae643e9b35fac1f629b13884b9923580b" prot="package" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>[static initializer]</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a351714d5339903e021be7a39e2adca53" prot="private" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>argument</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1aa34ff769fd5096cac91413650835d4f2" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>FunctionEvaluationException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1ac6a0fb990d010b9aa3919456478778d5" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>FunctionEvaluationException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a2bfa27be0ddaaaa744b9c2a7f83c5f89" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>FunctionEvaluationException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a74156e50005b1d7e243b7e797011fcf0" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>FunctionEvaluationException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a4d0053c292a35004a3d3d3362011cc3e" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>FunctionEvaluationException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1ae4059cbef0b27fabfc0973803fdea56d" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>FunctionEvaluationException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a28d8e7cbca775686a170215320c79b7a" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>getArgument</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1ace3e8d45b08371b293a371f939ca821a" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>getArguments</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1a3be3dc8a303a40c5ef8be3e703d7759c" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>getCause</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1af449321a8caeb2bd60c7aa7d6dd60510" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>getMessage</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1ab6959c6786a7152e04df1cb605e67d9f" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>getPattern</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1a512e05d6acfa2eef2bd019989dd768f6" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>MathException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1a4093c18d9b0ae89e0c050ae1aaae7cf4" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>MathException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1aaacc5d2460a751ee6037ca38c4ddd9c9" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>MathException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1ab0de029f791895c852cff2faf8d96c74" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>MathException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1ad12de7b99bf6951363e4fe3de9af6838" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>MathException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1a5ad7913bae4d6ddf1ef820ccd2a2577b" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>MathException</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1aafe7f38509d0d8ea5197f7200ed3a174" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>printStackTrace</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1a0baeac9b6bdd594c62c6759aae1d0d25" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>printStackTrace</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_math_exception_1ab8db75005a045db16c1b02868bc04f20" prot="public" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>printStackTrace</name></member>
      <member refid="classorg_1_1apache_1_1commons_1_1math_1_1_function_evaluation_exception_1a430040ad99f0d58fdcc50fe27516dac0" prot="private" virt="non-virtual"><scope>org::apache::commons::math::FunctionEvaluationException</scope><name>serialVersionUID</name></member>
    </listofallmembers>
  </compounddef>
</doxygen>
